### ISSUE TITLE
A few questions about data details

### ISSUE THREAD

**Issue by sookiemm** [AZZMM/CC-Diff#3]
A few questions about data details: First of all, thank the authors for their wonderful work. During my experiment, I have several questions related to data processing: 

1. Is the metadata.jsonl in the DIOR/Train and DIOR/Val folders the output of dior_txt_to_metajsonl.py or the output of LLM_Text2Layout.py (gpt_metadata.jsonl)? I see that gpt selects 5 train data similar to val data to get the layout, so can we assume that DIOR/Train/metadata.jsonl is generated by dior_txt_to_metajsonl.py, DIOR/Val/metadata.jsonl is generated by LLM_Text2Layout.py (gpt_metadata.jsonl), and when infer is performed after training, it is generated according to the val layout generated by gpt? 

2. When calculating yoloscore, does the author use DIOR/train + the images generated by the above infer as the training set, and gt val as the test set?

We look forward to your responses and once again thank the authors for their excellent work.

**Comment by AZZMM** [comment]
Thanks for the questions:
1. dior_txt_to_metajsonl.py outputs metadata.jsonl with gt layouts in both train and val folders. After that, you can use LLM_Text2Layout.py to generate gpt_metadata.jsonl with new layouts through LLM(e.g. GPT-4o). The new layouts are inferred according to the image captions in metadata.jsonl through in-context learning. This approach ensures that synthesizing a large number of training images along with annotations is not limited by the number of GT layouts.
2. Only gt images in DIOR are used for yoloscore metric, gt train set is used for training yolo model. Synthesized images are used in trainability experiments.

We will supplement the detailed explanation soon.

**Comment by sookiemm** [comment]
Sorry about the confusion between **YoloScore** and **Trainability** in the previous description. 

Exactly, most of my queries are for Trainability. I found out from your infer_dior.py: data_path = 'path_to_data/DIOR/val', so can I assume that the table2 results were trained with gt train + generated images, tested on gt val, and the generated images above were generated from the GT Val Layout (set by ref6).? However, such generation would be greatly limited by the quantity and quality of the gt layout, so you also tried LLM-based layout generation(LLM_Text2Layout.py) and got tabel6 (whose title specifically said "Layout generated using GPT-4o").

**Comment by AZZMM** [comment]
In the trainability experiments, the sythesized images are generated with GT train layouts directly which can refer to "Trainability" part in Section 5.2.2 in the preprint, "we double the training samples using layout-based synthesis and assess detection accuracy with the expanded dataset". The ratio of sythesized images to real images is 1:1.

Besides that, we conduct experiment shown in Table 6 to demonstrate the CC-Diff's adaptability to LLM-generated layout conditions. Detailed explanation can be found in "LLM-assisted T2I" part in Section 5.4.

**Comment by sookiemm** [comment]
Got it, thanks so much for the detailed explanation!
